src/AppKernel.php

```diff
@@ -46,14 +46,23 @@ class AppKernel extends Kernel
     private $routingFile = null;
 
     /**
-     * @param string $cachePrefix
+     * {@inheritDoc}
      */
-    public function __construct($cachePrefix)
+    public function __construct(/*string $environment, bool $debug*/)
     {
-        parent::__construct($cachePrefix, true);
-        $this->cachePrefix = $cachePrefix;
-        $this->addBundle(FrameworkBundle::class);
+        $args = \func_get_args();
+
+        if (1 === \func_num_args()) {
+            trigger_deprecation('nyholm/symfony-bundle-test', '1.9', 'The signature of the "%s($cachePrefix)" constructor is deprecated, use the constructor with 2 arguments: "string $environment, bool $debug".', self::class);
```

Comment:
I also thought about just using `var_dump` or something else, cause the trigger_deprecation doesn't print anything out so it will never be visible for the developer without a prober error handler registered.